AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/FragmentArgHandler.java

```diff
@@ -67,12 +86,18 @@ public void process(Element element, EFragmentHolder holder) {
 			argKey = fieldName;
 		}
 
-		TypeMirror actualType = codeModelHelper.getActualType(element, holder);
+		DeclaredType enclosingClassType;
+		if (param.getKind() == ElementKind.PARAMETER) {
```

Comment:
This kind of code is duplicated (in various ways) in several `handlers`. Is there a way to put this code in `InjectHelper` ?